### ISSUE TITLE
where can I see all property of file which  ionic.config.json?

### ISSUE THREAD

**Issue by Lyfei** [ionic-team/ionic-docs#2083]
Only a few properties are mentioned in the document
<img width="684" alt="image" src="https://user-images.githubusercontent.com/18715528/137063554-8c800e37-e40c-4f5e-a346-7eff399fd75e.png">

**Comment by amandaejohnston** [comment]
Thank you for the issue. A full list of the properties available in the config interface has since been added here: https://ionicframework.com/docs/angular/config#ionicconfig Closing this out.

**Comment by jonz94** [comment]
Hi,

I think the original issue is about the config of `@ionic/cli` (https://ionicframework.com/docs/cli/configuration#project-configuration-file), not the config in https://ionicframework.com/docs/angular/config#ionicconfig.

Should we consider re-open this issue? Thanks!

**Comment by amandaejohnston** [comment]
Yes, good catch. The `hooks` object is covered per the comment in the example JSON, but we should add or link to details about the `integrations` object.
